config/settings/production.py

```diff
@@ -201,7 +201,7 @@
 # ------------------------------------------------------------------------------
 OPENSEARCH_DSL = {
     "default": {
-        "hosts": [env("OPENSEARCH_HOST")],
+        "hosts": [env("OPENSEARCH_HOSTS")],
```

Comment:
why change?

Reply:
https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/
it was experimenting, to check